src/PIL/WmfImagePlugin.py

```diff
@@ -19,21 +19,15 @@
 # http://wvware.sourceforge.net/caolan/index.html
 # http://wvware.sourceforge.net/caolan/ora-wmf.html
 
-from __future__ import print_function
-
 from . import Image, ImageFile
 from ._binary import i16le as word, i32le as dword, si16le as short, si32le as _long
-from ._util import py3
 
 # __version__ is deprecated and will be removed in a future version. Use
 # PIL.__version__ instead.
 __version__ = "0.2"
 
 _handler = None
 
-if py3:
-    long = int
```

Comment:
Why remove this?

Reply:
`long` isn't used in this file.

(`_long` is used, which is imported from `._binary`.)

https://github.com/python-pillow/Pillow/blob/master/src/PIL/WmfImagePlugin.py